package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "endpoint_generator",
+  "version": "0.1.0",
+  "main": "index.js",
+  "repository": "git@git.endgames.local:ckim/endpoint_generator.git",
+  "author": "Chan Kim <ckim@endgame.com>",
+  "license": "MIT",
+  "scripts": {
+    "build": "tsc --downlevelIteration",
+    "start": "node bin/doc_gen",
+    "test": "jest --verbose"
+  },
+  "bin": {
+    "endpoint_generator": "bin/endpoint_generator.js"
```

Comment:
Are you missing files? I don't see `bin/endpoint_generator.js` in this pull request. See Chan's latest - https://git.endgames.local/ckim/endpoint_generator